py_mind_memo/dialogs.py

```
@@ -0,0 +1,81 @@
+
+    def show(self):
+        # 画面中央に配置
+        self.update_idletasks()
+        try:
+            parent = self.master
+            x = parent.winfo_x() + (parent.winfo_width() - self.winfo_reqwidth()) // 2
+            y = parent.winfo_y() + (parent.winfo_height() - self.winfo_reqheight()) // 2
+            self.geometry(f"+{x}+{y}")
+        except (AttributeError, tk.TclError) as e:
+            import logging
+            logging.getLogger(__name__).warning("Dialog centering failed: " + str(e))
```

Ruff (0.15.7)

[warning] Logging statement uses `+`

(G003)